torch/masked/maskedtensor/core.py

```diff
@@ -0,0 +1,556 @@
+                raise ValueError("__torch_dispatch__, {func}: expected args[0] to be a tensor")
+            mt = args[0]
+            if not is_masked_tensor(mt):
+                mt = MaskedTensor(mt, torch.ones_like(mt).bool())
+            if mt.is_sparse_csr():
+                return mt
+            new_mask = func(mask)
+            new_data = data.sparse_mask(new_mask)
+            return MaskedTensor(new_data, new_mask)
+        if func in [torch.ops.aten._to_dense]:
+            _check_args_kwargs_length(args, kwargs, f"__torch_dispatch__, {func}", len_args=1, len_kwargs=0)
+            if not torch.is_tensor(args[0]):
+                raise ValueError("__torch_dispatch__, {func}: expected args[0] to be a tensor")
+            mt = args[0]
+            if not is_masked_tensor(mt):
+                mt = MaskedTensor(mt, torch.ones_like(mt).bool())
+            new_data = func(data)
+            new_mask = func(mask)
+            return MaskedTensor(new_data, new_mask)
+        if func is torch.ops.aten._indices:
+            # Assumes data is sparse
+            _check_args_kwargs_length(args, kwargs, f"__torch_dispatch__, {func}", len_args=1, len_kwargs=0)
+            return MaskedTensor(data.indices(), torch.ones_like(data.indices()).bool())
+        if func is torch.ops.aten._values:
+            # Assumes data is sparse
+            _check_args_kwargs_length(args, kwargs, f"__torch_dispatch__, {func}", len_args=1, len_kwargs=0)
+            data = get_data(args[0]).values()
+            return MaskedTensor(data, torch.ones_like(data).bool())
+        if func is torch.ops.aten._sparse_coo_tensor_with_dims_and_tensors:
+            new_args = list(args)
+            if is_masked_tensor(args[-1]):
+                new_args[-1] = args[-1]._masked_data
+            if is_masked_tensor(args[-2]):
+                new_args[-2] = args[-2]._masked_data
+
+            new_data = func(*new_args, **kwargs)
+            new_args[-1] = torch.ones_like(new_args[-1])
+            new_mask = func(*new_args, **kwargs).bool()
+
+            return MaskedTensor(new_data, new_mask)
+        msg = (
+            f"{func.__name__} is not implemented in __torch_dispatch__ for MaskedTensor.\n"
+            "If you would like this operator to be supported, please file an issue for a feature request at "
+            "https://github.com/pytorch/maskedtensor/issues with a minimal reproducible code snippet.\n"
+            "In the case that the semantics for the operator are not trivial, it would be appreciated "
+            "to also include a proposal for the semantics."
+        )
+        warnings.warn(msg)
+        return NotImplemented
+
+    def __lt__(self, other):
+        if is_masked_tensor(other):
+            return MaskedTensor(get_data(self) < get_data(other), get_mask(self))
+        return MaskedTensor(get_data(self) < other, get_mask(self))
+
+    def to_tensor(self, value):
+        return get_data(self).masked_fill(~get_mask(self), value)
```

Comment:
I don't think there's ever a reason to call `get_data(self)`/`get_mask(self)`, is there? `self.get_data()/get_mask()` avoid the redundant type checks and  seem more idiomatic

Reply:
Yeah, I agree!